pghoard/wal.py

```diff
@@ -42,107 +44,156 @@ class WalBlobLengthError(ValueError):
     """WAL blob is shorter than the WAL header"""
 
 
-WalHeader = namedtuple("WalHeader", ("version", "timeline", "lsn", "filename"))
+WalHeader = namedtuple("WalHeader", ("version", "lsn"))
 
 
-def read_header(blob):
-    if len(blob) < WAL_HEADER_LEN:
-        raise WalBlobLengthError(
-            "Need at least {} bytes of input to read WAL header, got {}".format(WAL_HEADER_LEN, len(blob))
+class LSN:
```

Comment:
It would be nice if all these properties/methods would have at least a one line docstring.

Reply:
Ok, will do !